TEMPLATE.md

```diff
@@ -0,0 +1,428 @@
+
+# Writing Your Own Templates
```

Comment:
I don't think `template.md` is a good idea, simply because I don't think it will be used.

I have sympathy with the idea of being able to read it in GitHub, but I think there are alternatives (hex docs, iex help, and so on) that do the job, too.

Reply:
Removing